testscommon/chainHandlerStub.go

```diff
@@ -17,6 +17,14 @@ type ChainHandlerStub struct {
 	GetCurrentBlockRootHashCalled          func() []byte
 	SetFinalBlockInfoCalled                func(nonce uint64, headerHash []byte, rootHash []byte)
 	GetFinalBlockInfoCalled                func() (nonce uint64, blockHash []byte, rootHash []byte)
+
+	currentBlockHeader   data.HeaderHandler
```

Comment:
this tends to become a mock. might create a new structure called chainHandlerMock with these changes

Reply:
Reverted changes, added mock.

(not used anymore / yet, since `nodeLoadAccounts_test.go` has changed, but added anyway).